src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -37,6 +37,8 @@
 /* Internal Class Implementation                                        */
 /************************************************************************/
 
+#define ROUNDNESS_EPSILON 1.0f
```

Comment:
We must standardize a single definition in the roundcorner PR; otherwise, I suggest removing the constant because 1.0f itself already looks clear and deterministic, not like other magic numbers.

Reply:
this one is needed in the tvgLottieBuilder.cpp and in the tvgLottieProperty.h so I'd like to keep it as a const in tvgLottieProperty.h